.github/workflows/ci-build.yml

```diff
@@ -50,30 +72,32 @@ jobs:
           - ember-lts-4.12
           - ember-lts-5.4
           - ember-lts-5.8
-          - ember-release
-          - ember-beta
-          - ember-canary
+          # All of these are failing with the waiter tests (needs to be debugged separately)
```

Comment:
Can you open an issue so this isn't forgotten?

Reply:
there is this one: https://github.com/emberjs/ember.js/issues/20724

Reply:
https://github.com/emberjs/ember-test-helpers/issues/1465